client/my-sites/upgrades/domain-management/list/item.jsx

```diff
@@ -24,7 +25,7 @@ const ListItem = React.createClass( {
 
 					<div className="domain-management-list-item__meta">
 						<span className="domain-management-list-item__type">{ this.getDomainTypeText() }</span>
-
+						{ this.props.domain.type !== 'WPCOM' ? this.isDomainExpired( this.props.domain ) : null }
```

Comment:
@umurkontaci Can definitely change the name. Makes total sense. With `isRegisteredDomain`, that only returns true for registered domains (not mapped, subdomains or site redirects) if I'm reading [`lib/domains/index.js`](https://github.com/Automattic/wp-calypso/blob/f6aa008b7dd1a8c2357626966d1ff6f67c806c0c/client/lib/domains/index.js#L140) correctly. I can use the companion [`isMapped`](https://github.com/Automattic/wp-calypso/blob/f6aa008b7dd1a8c2357626966d1ff6f67c806c0c/client/lib/domains/index.js#L122) and [`isSubdomain`](https://github.com/Automattic/wp-calypso/blob/f6aa008b7dd1a8c2357626966d1ff6f67c806c0c/client/lib/domains/index.js#L109), but that still leaves site redirects. Feels simpler to just filter out `domain.type === WPCOM`. Thoughts?